python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -829,7 +829,7 @@ def assembleMetadata(self, coaddExposure, tempExpRefList, weightList):
                            for tempExpRef in tempExpRefList]
         numCcds = sum(len(tempExp.getInfo().getCoaddInputs().ccds) for tempExp in tempExpList)
 
-        coaddExposure.setFilter(tempExpList[0].getFilter())
+        coaddExposure.setFilterLabel(tempExpList[0].getFilterLabel())
```

Comment:
I just realized that this will give coadds a physical filter (in general). I'm fine with that (especially if all the inputs use exactly the same filter), but I think it was argued at some point that coadds should be by band?

Reply:
I think that we do need to use band.  For HSC this shows up in e.g. i and i2 physical filters;  for Rubin it'll be ITL and E2V CCDs, which have different effective bandpasses.

Reply:
Sorry, just to be clear, are you saying the code should _remove_ any physical filter in the `FilterLabel` it assigns to `coaddExposure`? A band will be present either way.

Reply:
@RobertLuptonTheGood , following up on @kfindeisen 's comment: should coadds *only* have `band`, or can they have both `band` and `physical` (as this change implements)?

I also don't understand your comment about different CCDs: that has nothing to do with the physical/band FilterLabels.

Reply:
> Sorry, just to be explicit, are you saying the code should remove any physical filter in the `FilterLabel` it assigns to `coaddExposure`? A band should be present either way.

This isn't something I'd considered before now, but I do think it would probably be best to clear the physical filter label from any coadd image, since it isn't always going to be meaningful and we don't have a clear use case for it.  

I think I probably would continue to use `physical_filter` in coaddInputs, however, as that will convey more information, and I can't think of any case where a single `coaddInput` record might have an ambiguous `physical_filter`.

I think we're already propagating ITL vs E2V via the `Detector` object that's also attached to the coaddInputs records (we certainly could be; there is definitely a slot for it).  And we're already propagating the combined throughput via the attached `TransmissionCurve` (which are actually combine quantitatively in coaddition, unlike the more descriptive `FilterLabel` and `Detector` information).